style.css

```diff
@@ -0,0 +1,342 @@
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+
+}
+a{
+	color: white;
+	text-decoration: none;
+}
+a:hover{
+	color: rgba(5, 171, 101, 1);
+}
+#connect{
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	font-size: 20px;
+	width: 100px;
+}
+#githubLink{
+	height: 35px;
+	width: 35px;
+
+}
+#github{
+	width: 35px;
+	height: 35px;
+
+}
+#navbar{
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: space-around;
+
+	background-color: rgba(59, 91, 105, .8);
+	border: 2px solid	rgba(5, 171, 101, 1);
+
+	height: 30px;
+	width: 500px;
+	margin: 10px;
+	border-radius: 10px;
+}
+#login{
+	color: black;
+	margin: 5px;
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+}
+#landingMain{
+	display: flex;
+	flex-direction: row;
+	align-items: flex-start;
+	justify-content: space-around;
+	background-color: rgba(32, 97, 204, .6);
+	border: 3px solid rgba(180, 35, 35, 1);
+	border-radius: 10px;
+	padding: 10px;
+	width: 1000px;
+	margin: 10px;
+}
+#genInfo{
+	color: white;
+	width: 500px
+	padding: 5px;
+}
+#pricing{
+	color: white;
+	padding: 5px;
+
+}
+#updateHistory{
+	color: white;
+	list-style: none;
+	padding: 5px;
+	margin: 5px;
+	box-sizing: border-box;
+
+}
+#updateForm{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+.formSpot:hover{
+	background-color: rgba(186, 171, 130, .7);
+	border: 2px solid rgba(166, 43, 39, 1);
+	color: black;
+}
+#inptTntTitle{
+	color: white;
+	margin: 5px;
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+}
+#remRmmt{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+#remove-roomate-button:hover{
+	background-color: rgba(186, 171, 130, .7);
+	border: 2px solid rgba(166, 43, 39, 1);
+	color: black;
+	text-align: center;
+	font-size: 20px;
+	height: 30px;
+	border-radius: 5px;
+}
+#remBill{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+#remove-bill-button:hover{
+	background-color: rgba(186, 171, 130, .7);
+	border: 2px solid rgba(166, 43, 39, 1);
+	color: black;
+	text-align: center;
+	font-size: 20px;
+	height: 30px;
+	border-radius: 5px;
+}
+#fields{
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+}
+#monthName{
+	margin: 5px;
+	font-size: 20px;
+	height: 30px;
+	width: 250px;
+	border-radius: 5px;
+}
+.submit{
+	background-color: rgba(186, 171, 130, .7);
+	border: 2px solid rgba(166, 43, 39, 1);
+	color: black;
+	text-align: center;
+	font-size: 20px;
+	margin: 10px;
+	height: 30px;
+	width: 300px;
+	border-radius: 5px;
+}
+.submit:hover{
+	background-color: rgba(59, 91, 105, .8);
+	border: 2px solid	rgba(5, 171, 101, 1);
+	color: white;
+}
+#nr{
+	width: 200px;
+}
+#nb{
+	width: 100px;
+}
+#roommateList{
+	list-style: none;
+	color: white;
+	margin: 5px;
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+}
+#roommateList.li{
+	color: white;
+	margin: 5px;
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+}
+#rmmtTitle{
```

Comment:
There a quite a few reused property/values in your CSS, as well as many ID selectors. You can refactor this to utilize classes, with repetitive rules, and then apply those classes to your tags. It's a refactor but it'll clean up the CSS quite a bit.